src/components/TransactionPreview/HistoryPreview.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { ethers } from 'ethers';
+
+// Components
+import Card from '../Card';
+import { ClickableText, Text } from '../Text';
+import { CombinedRoundedImages, RoundedImage } from '../Image';
+
+// Utils
+import { formatAmountDisplay, humanizeHexString, copyToClipboard, getTypeOfAddress } from '../../utils/common';
+import { Chain, CHAIN_ID, klimaAsset, nativeAssetPerChainId, supportedChains } from '../../utils/chain';
+import { Theme } from '../../utils/theme';
+
+// Constants
+import { TRANSACTION_BLOCK_TYPE } from '../../constants/transactionBuilderConstants';
+import { CHAIN_ID_TO_NETWORK_NAME } from 'etherspot/dist/sdk/network/constants';
+
+import moment from 'moment';
+
+// Hooks
+import { useEtherspot } from '../../hooks';
+
+// Types
+import { ICrossChainAction, SendAssetActionPreview } from '../../types/crossChainAction';
+
+//Icons
+import { RiExternalLinkLine } from 'react-icons/ri';
+import { LuFuel } from 'react-icons/lu';
+
+const MainWrapper = styled.div`
```

Comment:
We usually put the styles at the bottom of the file, for consistency

Reply:
Changes done